tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md

```diff
@@ -0,0 +1,155 @@
+
+
+## Boot dari DVD
+Duration: 0:03
+
+Sangat mudah memasang Ubuntu dari DVD. Inilah yang perlu kamu lakukan:
+1. Letakkan Ubuntu DVD di drive optik/DVD kamu.
+1. Restart komputer kamu.
+
+Tunggu beberapa saat, setelah desktop termuat, kamu akan melihat jendela selamat datang. Dari sini kamu bisa memilih bahasa yang berada di list sebelah kiri dan memilih di antara menginstal Ubuntu secara langsung, atau mencoba desktop terlebih dahulu (Jika kamu menyukai apa yang kamu lihat, kamu bisa memasang Ubuntu dari mode ini juga).
+
+
+![screenshot](https://assets.ubuntu.com/v1/dfc572f0-download-desktop-install-ubuntu-desktop_1.jpg)
+
+Jika kamu tidak mendapatkan menu ini, baca [panduan booting dari DVD](https://help.ubuntu.com/community/BootFromCD?_ga=2.102380610.2115462233.1496186978-1155966827.1485186360) untuk info lebih lanjut.
+
+## Boot dari Drive USB
+Duration: 0:02
+
+Kebanyakan komputer akan boot dari USB secara otomatis. Masukkan saja flash disk USB dan nyalakan komputer kamu atau restart. Kamu akan melihat jendela selamat datang seperti kita lihat sebelumnya di langkah 'Install dari DVD', meminta kamu untuk memilih bahasa dan menginstal Ubuntu secara langsung, atau mencoba desktop terlebih dahulu.
+
+Jika komputer kamu tidak otomatis boot dari USB, coba tekan `F12` ketika komputer mulai menyala. Di banyak mesin ini akan mengizinkan kamu untuk memilih USB dari menu boot spesifik sistem.
+
+positif
+: F12 adalah kunci paling umum untuk menampilkan boot menu, tetapi gabungan Escape dan F2 bisa menjadi alternatif. Jika kamu tidak yakin, lihat pesan singkat ketika memulai sistem - ini akan memberikan informasi kepada kamu mengenai kunci yang harus ditekan untuk menuju boot menu.
+
+## Persiapan memasang Ubuntu
+Duration: 0:01
+
+Setelah memilih untuk memasang Ubuntu dari jendela selamat datang, kamu akan ditanyakan mengenai pembaharuan dan perangkat lunak pihak ketiga.
+
+- Kami sarankan untuk mengaktifkan kedua `Unduh pembaruan` dan `Pasang perangkat lunak pihak ketiga`.
+- Tetaplah terhubung ke internet jadi kamu bisa mendapatkan pembaharuan terakhir ketika memasang Ubuntu.
+- Jika kamu tidak terkoneksi ke internet, kamu akan ditanyakan untuk memilih jaringan nirkabel, jika tersedia. Kami sarankan agar terkoneksi ketika melakukan pemasangan jadi kami bisa memastikan mesin mutakhir.
+
+
+![screenshot](https://assets.ubuntu.com/v1/3bbb0e35-download-desktop-install-ubuntu-desktop_2.jpg)
+
+## Alokasi ruang drive
+Duration: 0:01
+
+Gunakan kotak centang untuk memilih apakah kamu ingin Memasang Ubuntu berdampingan dengan sistem operasi lain, menghapus sistem operasi yang Anda dan menggantikannya dengan Ubuntu, atau — jika kamu adalah pengguna tingkat lanjut  — pilih opsi  ’**Something else**’.
```

Comment:
sori, review saya sebelumnya ternyata ada typo
delete your existing operating system and replace it with Ubuntu -> menghapus sistem operasi yang ada dan menggantikannya dengan Ubuntu